### DISCUSSION TITLE
stylelint vscode extension 통합 방법

### DISCUSSION THREAD

**Discussion by uraflower** [KakaoFunding/front-end#39] [Announcements]
## 시작에 앞서
`stylelint` 또한 `eslint`, `prettier` 처럼 공식적으로 `vscode extension`을 제공합니다.
이것을 활용하면 매번 명령을 입력하지 않아도 실시간으로 lint 오류를 잡을 수 있습니다.

그런데, **`stylelint extension` 기본 설정에는 `scss`가 빠져 있습니다.**

> #### _예시_
> ![image](https://github.com/KakaoFunding/front-end/assets/82873315/63c34b1d-d2d8-4e10-aa00-0d65c63d8fc2)
> _색상 지정 시 이름으로 지정할 수 없는 규칙이 있음에도, 오류가 나지 않는 모습_

그래서 설정하는 방법을 알려드립니다.

## 통합 및 설정

1. `vscode`에서 `stylelint` extension을 설치합니다.
2. `stylelint `extension 설정에 들어갑니다.
![image](https://github.com/KakaoFunding/front-end/assets/82873315/ef6380eb-05bd-482a-afcb-8d22d3ebbf9a)
3. `validate` 옵션에 `scss`를 추가합니다.
![image](https://github.com/KakaoFunding/front-end/assets/82873315/809d7ece-157c-4d3e-ab82-932620d70d4e)
4. 성공적으로 오류를 잡아줍니다.
![image](https://github.com/KakaoFunding/front-end/assets/82873315/da6f64cd-7293-4eaa-a11f-2fe9973d60b1)

**Comment by devkyoung2** [comment]
감사합니다~
